ChatCore/Models/MessagesRequest.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  MessagesRequest.swift
+//  STRVChatCore
+//
+//  Created by Daniel Pecher on 23.02.2021.
+//
+
+import Foundation
+
+public struct MessagesRequest {
+    public let messageId: EntityIdentifier
+    public let direction: LoadingDirection
+    public let count: Int
+    public let includeInResult: Bool
+    
+    public init(messageId: EntityIdentifier, direction: LoadingDirection, count: Int, includeInResult: Bool) {
```

Comment:
You can add default values for `count` (default page size) and `includeInResult`